game/game.py

```diff
@@ -64,17 +50,17 @@ def update_score(score, winner):
     return score
 
 
-def make_decision(user_choice, comp_choice):
+def get_decision(user_choice, comp_choice):
     """Returns the result of user's decision compared with computer's decision, exits the program if user prints
     exit."""
     winner = ""
+
     if user_choice == 'exit':
-        print(exit_program())
-        sys.exit()
+        return exit_program()
```

Comment:
again we don't want to return text, we need to return values where it's needed,
when we are exiting there is nothing to return, this change is wrong roll it back for now.